docs/content/faq.md

```diff
@@ -58,3 +58,37 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
 ## HedgeDoc fails executing migrations and does not start
 Unfortunately, older versions of HedgeDoc had some bugs regarding migrations and didn't always record that a migration was executed.  
 Have a look at the *[Troubleshooting Migrations](/guides/migration-troubleshooting/)* guide for more information.
+
+## Why does my interface look weird?
+
+If your interface looks like this
+
+![Broken HedgeDoc interface](./faq/interface_broken.png)
+
+instead of this
+
+![Not broken HedgeDoc interface](./faq/interface_okay.png)
+
+then you should check the following settings (or their corresponding environment variables if you e.g. use docker.):
+
+- `domain`
+- `protocolUseSSL`
+- `urlAddPort`
```

Comment:
Maybe just add the environment variables here too, as we recommend the docker setup and that uses them.

```suggestion
- `domain` (env: `CMD_DOMAIN`)
- `protocolUseSSL` (env: `CMD_PROTOCOL_USESSL`)
- `urlAddPort` (env: `CMD_URL_ADDPORT`)
```